article/app/services/dotcomponents/ArticlePicker.scala

```diff
@@ -106,8 +77,6 @@ object ArticlePicker {
         "isNotAGallery",
         "isNotLiveBlog",
         "isNotAMP",
-        "isNotInTagBlockList",
-        "isNotPaidContent",
```

Comment:
this might actually drop our support % a little because I think this is out-of-date - tag blocklist and paid content are legit things we should support (and are very close!)